### PR TITLE
Metadata widget

### PR DESCRIPTION
**Pull request by thecalcc**

### CODE REVIEW

**Comment by thecalcc** [comment]
Get  & vocabularies

```
            <li class="basic-list__item" ng-repeat="cv in cvs track by cv._id" ng-if="item[cv.schema_field] | mergeWords:'name' : cv._id">
                <label class="basic-list__item-label">{{ cv.display_name | translate }}</label>
                <div class="basic-list__item-data">
                        {{getLocaleName(item[cv.schema_field], cv._id)}}
                </div>
            </li>
```

**Comment by tomaskikutis** [comment]
* ~~bind onArticleChange~~
* dataApi.query<ILanguage>(  'languages',  // <------ check if already fetched in metadata
* make label optional for Switch component and allow passing custom id
* keep capitalization of labels so existing ones don't get discarded
* ~~Copyright label change (copyrightnotice, creditline)~~
* sd-html-preview and other directives

**Comment by tomaskikutis** [comment]
css file next to tsx
mergeArrayToString | rename and no conditional return >> move to vocabularies in api

getAllAnnotations - type argument

getLocaleName / getVocabularyItemLabel - pass arguments explicitly

be more careful with optional chaining - don't use it where it's not needed

File: scripts/api/vocabularies.ts [hunk 1]
```diff
@@ -29,19 +29,81 @@ function getVocabularyItemLabel(term: IVocabularyItem, item: IArticle): string {
...
+): Array<string> => {
+    let subjectMerged = [];
+
+    array.forEach((item) => {
+        const value = propertyName == null ? item : item[propertyName];
+
+        if (value) {
+            subjectMerged.push(value);
+
+            if ((schemeName?.length ?? 0) && item.scheme !== schemeName) {
```
Comment (tomaskikutis) [inline scripts/api/vocabularies.ts:52]: compare the number to something

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**